src/ServiceFabric/ServiceFabric/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 3.3.3
```

Comment:
Need to add the message in the `Upcoming Release` section instead of creating a new version.

Reply:
updated